src/components/MarkdownProvider/components/Usage.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { MD } from '@zendeskgarden/react-typography';
+
+export const Use: React.FC = props => (
+  <div>
+    <MD isBold>When to use</MD>
+    {props.children}
+  </div>
+);
+
+export const Misuse: React.FC = props => (
+  <div>
+    <MD isBold>When NOT to use</MD>
+    {props.children}
+  </div>
+);
+
+export const Usage = styled.div`
+  @media (min-width: ${p => p.theme.breakpoints.sm}) {
+    display: flex;
```

Comment:
@jzempel This is more personal curiosity. When I was tinkering with this, I threw some Grid usage against the wall and I was curious if we are open to that? I don't think it's blocking but I'd love to [get some space in between the columns so they don't butt up against eachother. ](https://share.getcloudapp.com/o0ugOp25)

I had something like this to [create space between and below](https://share.getcloudapp.com/WnuGXdxK): 
```
  display: grid;
  grid-gap: ${p => p.theme.space.base * 6}px;
  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
  margin: 0 0 ${p => p.theme.space.base * 9}px;
```
We can ofcourse do this with Flex (and prob should if this is a no-go).

Reply:
@rossmoody, what do you think about using Garden's `react-grid` package to achieve the layout? It would also address the spacing issue you've discovered. I see two benefits of this: 1) Full support for IE11 2) More Garden usage. Let me know what you think.

Reply:
Great solve 👍